documentation/guide-service-layer.asciidoc

```diff
@@ -6,32 +6,22 @@ toc::[]
 The service layer is responsible for exposing functionality made available by the link:guide-logic-layer.asciidoc[logical layer] to external consumers over a network via xref:protocol[technical protocols].
 
 == Types of Services
-We distinguish between the following types of services:
 
-* *External Services* +
-are used for communication between different companies, vendors, or partners.
-* *Internal Services* +
-are used for communication between different applications in the same application landscape of the same vendor.
-** *Back-end Services* +
-are internal services between Java back-end components typically with different release and deployment cycles (if not Java consider this as an external service).
-** *JS-Client Services* +
-are internal services provided by the Java back-end for JavaScript clients (GUI).
-** *Java-Client Services* +
-are internal services provided by the Java back-end for a native Java client (JavaFx, EclipseRcp, etc.).
+Before you start creating your services you should consider some general design aspects:
 
-The choices for technology and protocols will depend on the type of service. The following table gives a guideline for aspects according to the service types.
+* Do you want to create a https://en.wikipedia.org/wiki/RPC[RPC] service?
```

Comment:
This link is odd. It does not provide any value as it is not pointing to a concrete content side of wikipedia

Reply:
What is odd with wikipedia? This link helps if someone does not know what RPC stands for. What would you like to explain this?

Reply:
The link you provided does not help to understand what RPC stands for as you simply provide a generic link and wikipedia is asking you to select the content of interest. ;)

Reply:
Ouch, now I understand. Sorry, that was a C&P mistake. Should have been https://en.wikipedia.org/wiki/Remote_procedure_call
Will fix it asap.